common/djangoapps/pipeline_mako/templates/i18n_helpers.html

```diff
@@ -0,0 +1,31 @@
+<%page expression_filter="h"/>
```

Comment:
Thanks, @amangano-edx. It is much better to keep the i18n stuff together. 

Nit: I wonder why this needs to be in ``pipeline_mako``. This doesn't seem like pipeline-specific logic, so I wouldn't expect to see it live here. It seems that it would be simpler if this lived in i18n_helpers.py that had unit tests, rather than having the code be Mako-specific. It would also be great to have all the i18n logic live in one Django app, rather than be spread around the platform.  This is just a nit so you can choose to address it or not.